lib/pager.dart

```diff
@@ -445,6 +460,30 @@ class _PagerState<K, T> extends State<Pager<K, T>> with AutomaticKeepAliveClient
     return builder;
   }
 
+  Widget _buildContent(BuildContext context, ScrollController? scrollController,
+      PagingData<T> value) {
+    return renderOnlyWhenRemoteIsUpdated(
+      data: value,
+      successView: (data) => PagedListView(
+        itemBuilder: (context, index) {
+          final item = data[index];
+          return widget.itemBuilder?.call(item as T) ?? const SizedBox.shrink();
+        },
+        itemCount: data.length,
+        separatorBuilder: (ctx, index) =>
+            widget.separatorBuilder?.call(ctx, index) ??
+            const SizedBox.shrink(),
+        controller: scrollController ?? ScrollController(),
+        loadState: _states.append,
+        bottomLoadingIndicator: widget.bottomLoadingIndicator,
+      ),
+      errorView: (err) =>
+          widget.errorView?.call(err) ?? const SizedBox.shrink(),
+      loadingView: widget.loadingView,
+      emptyView: () => widget.emptyView?.call() ?? const SizedBox.shrink(),
+    );
```

Comment:
i don't think we should have this in Pager..  i think this should be done in a way where the user can supply this themselves and not directly part of Pager

Reply:
> i don't think we should have this in Pager.. i think this should be done in a way where the user can supply this themselves and not directly part of Pager

I've updated the implementation